deploy/operator/ztp/none_platform_utils.sh

```diff
@@ -71,7 +71,7 @@ function setup_libvirt_dns()
 
 function open_firewall_ports()
 {
-    for port in 6443 22623 80 81 443 ; do
+    for port in 6443 22623 22624 80 81 443 ; do
```

Comment:
Where did you see that 22624 is being used?

Reply:
By testing.  It doesn't work otherwise.

Reply:
There are references to this port.  For example 
https://access.redhat.com/solutions/5709711

and 

https://bugzilla.redhat.com/show_bug.cgi?id=1759338

Reply:
yes, but who does it? our code does not access 22624, am I wrong? What is this file (none_platfrom_utils.sh) is even about?

Reply:
It seems to me it is used to get day2 ignition.

Reply:
machine-config listens on this port

Reply:
_[root@ostest-extraworker-0 core]# netstat -lnp | grep 22624
tcp6       0      0 :::22624                :::*                    LISTEN      7318/machine-config 
[root@ostest-extraworker-0 core]# ps -ef | grep 7318
1000320+    7318    7305  0 Feb27 ?        00:00:02 /usr/bin/machine-config-server start --apiserver-url=https://api-int.assisted-test-cluster.redhat.com:6443
root     1806154 1804595  0 08:55 pts/0    00:00:00 grep --color=auto 7318
_

Reply:
none_platform_utils.sh is a utilities file for none platform operations. It performs 3 general operations:
- Open firewall ports for LB
- Setup and run LB
- Make libvirt DNS point to LB